File: Python/Easy/تكرار النص.py

# Creator: OmarAfet
# https://profile.satr.codes/OmarAfet/public/overview
# https://github.com/OmarAfet

def word_repeat(word: str, n: int) -> str:
    if (' ' in word) == False:
        word = word + ' '
        x = word * n
        return x[:-1]
    else:
        x = word * n
        return x[:-1]

File: challenges/JavaScript/Easy/تكرار النص.js
/*

Creator: OmarAfet
https://profile.satr.codes/OmarAfet/public/overview
https://github.com/OmarAfet

*/

function word_repeat(word, n) {
	x = "";
	for (let i = 0; i < n; i++) {
		x += `${word} `;
	}
	return x.slice(0, -1);
}
